src/main.rs

```diff
@@ -524,6 +524,19 @@ fn run() -> Result<(), failure::Error> {
                         .long("env")
                         .takes_value(true)
                 )
+                .arg(
+                    Arg::with_name("tunnel_port")
+                        .help("port to accept tail log requests")
+                        .short("p")
+                        .long("port")
```

Comment:
why does this one get `port` and the other one gets `metrics`?

i think i'd make them `tunnel_port` and `metrics_port` without a `.short` for either of them

Reply:
honestly i did it because i expect that users are way less likely to set the metrics port and also because i want to deprecate that eventually in favor of a less hairy method for extracting the port

Reply:
i feel like if they're setting one they'll set both - i'm not super hung up on this one but it feels strange for one to be `--port` when they are both ports

Reply:
again, wanna deprecate metrics. i could give it `--metrics-port` instead of just metrics but i'm pretty set on the primary being just `--port`